src/confluent_kafka/src/Admin.c

```diff
@@ -2835,6 +3199,121 @@ static PyObject *Admin_c_DescribeConsumerGroupsResults_to_py(
         return NULL;
 }
 
+static PyObject *Admin_c_ScramMechanism_to_py(rd_kafka_ScramMechanism_t mechanism){
+        PyObject *result = NULL, *value_dict = NULL;
+        PyObject *args = NULL;
+        PyObject *ScramMechanism_type;
+        value_dict = PyDict_New();
+        cfl_PyDict_SetInt(value_dict, "value",(int) mechanism);
+        args = PyTuple_New(0);
+        ScramMechanism_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                  "ScramMechanism");
+        result = PyObject_Call(ScramMechanism_type, args, value_dict);
+        Py_DECREF(value_dict);
+        Py_DECREF(ScramMechanism_type);
```

Comment:
Forgot to Py_DECREF for args.

```suggestion
        Py_DECREF(args);
        Py_DECREF(value_dict);
        Py_DECREF(ScramMechanism_type);
```